addons/old-studio-layout/addon.json

```diff
@@ -1,16 +1,11 @@
 {
   "name": "Old studio layout",
-  "description": "Reverts the layout of the studio page back to the one used before the studio update or the one previously available as part of the Scratch 2.0 \u2192 3.0 addon.",
+  "description": "Reverts the layout of the studio page back to the one used before the studio update or the one previously available as part of the Scratch 2.0 \u2192 3.0 addon. The size of most elements is not changed. If you want to see more projects and curators at once, use the \"more items per row\" addon.",
```

Comment:
I think this one should stay as a notice. We don't have "see this other addon" functionality yet, but if we do at some point, the list of related addons won't be listed on the description, but probably somewhere more visible. A notice is closer to that than simply appending to the description.

Reply:
Good point - maybe we should add an "info" type message that doesn't stand as much? The color shouldn't show off too much, and the info text should be the same size as the description text. What do you think?

Reply:
Like this?
![image](https://user-images.githubusercontent.com/51849865/175766435-ba6abdb2-90b9-46f3-84ba-5664afcf0224.png)
![image](https://user-images.githubusercontent.com/51849865/175766448-1b97b961-4cce-4da8-bd52-368fe05007d2.png)

Reply:
Perhaps the border in dark mode should be a little brighter to be consistent with the dark mode notice border and the light mode info message?

I agree that this is a good transient solution, though.